packages/core/src/controllers/relayer.ts

```diff
@@ -221,15 +264,20 @@ export class Relayer extends IRelayer {
     this.provider.on(RELAYER_PROVIDER_EVENTS.error, (err: unknown) =>
       this.events.emit(RELAYER_EVENTS.error, err),
     );
+
+    this.events.on(RELAYER_EVENTS.connection_stalled, async () => {
```

Comment:
Can probably simplify to

```
this.events.on(RELAYER_EVENTS.connection_stalled, this.restartTransport)
```

Reply:
Using that approach leads to undefined errors unless this.restartTransport is used as arrow fx which looks weird as it breaks the standard for other fx
e.g. 
```
private restartTransport = () => {}
```

will prolly refactor at a later stage